### PR TITLE
[MINOR] ui improvement

### PR DESCRIPTION
**Pull request by baekhoseok**
### What is this PR for?
* configuration sorting
* remove tooltip in job menu
* change repository infomation icon

### What type of PR is it?
Improvement

### Screenshots (if appropriate)

* configuration sorting

before

after
configuration_menu

* remove tooltip in job menu

before

after

* change repository information icon

before 

after

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? yes

### CODE REVIEW

**Comment by jongyoul** [comment]
LGTM. It's simple and clean. :-)

**Comment by soralee** [comment]
Looks good to me 
It's very clear :)

**Comment by anthonycorbacho** [comment]
Change looks great to me, that you for your contribution.
Awesome!

**Comment by AhyoungRyu** [comment]
@baekhoseok Thanks for your contribution and looks nice to me! 

Probably it's a nitpick, how about keeping consistency for that two buttons in interpreter setting page?

Here are two suggestions.
##### 1. Icons with tooltips and no button border

option_1

##### 2. Icons with button border and no tooltip

option_2

What do you think? :)

**Comment by baekhoseok** [comment]
@jongyoul @soralee @anthonycorbacho @AhyoungRyu  Thank you for review ^^
@AhyoungRyu Thank you for suggestion. I fixed it.

**Comment by AhyoungRyu** [comment]
@baekhoseok Tested and it looks good! You chose second option :)
screen shot 2016-10-21 at 12 46 19 pm
This is the current status that I build locally.

But one wired thing is I can't build with `./grunt build` in this branch. (no problem in master now)
screen shot 2016-10-21 at 12 42 13 pm

IDK what the exact cause is, I fixed this with rebasing. 
So could you rebase with below steps?

```
$ git checkout master
```

if you didn't set upstream yet (or you can skip this step)

```
$ git remote add upstream https://github.com/apache/zeppelin
```

rebase from master
```
$ git fetch upstream
$ git pull upstream master
$ git checkout minor_ui_fix
$ git rebase master
$ git push origin -f minor_ui_fix
```

**Comment by minahlee** [comment]
@AhyoungRyu the issue you raised is not caused by this pr but #1501. Can you try after clean bower cache(`bower cache clean`) and remove `bower_components` directory under `zeppelin-web`?

File: zeppelin-web/src/app/interpreter/interpreter.html [hunk 1]
```diff
@@ -19,11 +19,11 @@ <h3 class="new_h3">
...
         <div class="pull-right" style="margin-top:10px;">
-          <span style="cursor:pointer;margin-right:4px;"
-                ng-click="showRepositoryInfo = !showRepositoryInfo"
-                tooltip-placement="bottom" tooltip="Repository information">
-            <i class="fa fa-cog" ng-style="{color: showRepositoryInfo ? '#3071A9' : 'black' }"></i>
-          </span>
+          <button class="btn btn-default btn-sm"
+                  ng-click="showRepositoryInfo = !showRepositoryInfo">
+            <i class="fa fa-database" ng-style="{color: showRepositoryInfo ? '#3071A9' : 'black' }"></i>
+            Repository Information
```
Comment (minahlee) [inline zeppelin-web/src/app/interpreter/interpreter.html]: How do you think about changing `Repository Information` to `Repository`? For me, it seems like one button taking too much of space.

**Comment by minahlee** [comment]
@baekhoseok Thank you for the contribution! Do you mind to update `docs/assets/themes/zeppelin/img/ui-img/interpreter_menu.png` again?

**Pull request closed without merging**